### DISCUSSION TITLE
How to prevent new sign ups?

### DISCUSSION THREAD

**Discussion by ml-airflow** [illacloud/illa-builder#2015] [Q&A] [answered]
Sorry if this has been asked before, but it seems anyone can sign up and go into everything by default on the docker version; we use that version for some internal testing, but we are to give out logins; people shouldn't be able to sign up. How can I switch it off?

**Comment by wenzhuo2022** [comment, marked as answer]
Hello, you can set it via this way:  [Member list] > the [...] button on the upper right corner > Disable sign-up
